tests/test_fifo_scheduler.py

```diff
@@ -0,0 +1,146 @@
+
+
+import os
+import pprint
+import pytest
+
+import threading as mt
+
+import radical.utils           as ru
+import radical.pilot           as rp
+import radical.pilot.constants as rpc
+
+from radical.pilot.agent.scheduler.continuous_fifo import ContinuousFifo
```

Comment:
This module no longer exists, it should be `from radical.pilot.agent.scheduler.continuous_ordered import ContinuousOrdered`